run_tests.py.in

```diff
@@ -301,12 +280,20 @@ def clean_test_tree(path):  # type: (str) -> None
 def filter_tests(path):  # type: (str) -> list
     """Filter and sort tests in ascending order."""
     test_files = [str(f) for f in Path(path).rglob('*.re')]
-    test_files.sort()
     return test_files
 
 
-def run_pack(tests_pack):  # type: (list[str]) -> None
-    """Run tests pack."""
+# Pattern compilation in a loop is not a good idea.
+# Thus, we prepare patterns once.
+re2c_pattern = re.compile(r'.*re2c[ \t]*(.*)$')
+type_header_pattern1 = re.compile(r'--type-header[ \t]*')
+type_header_pattern2 = re.compile(
+    r'.*--type-header[ \t]*([^ \t]*).*'
+)
```

Comment:
Nitpic: Could you please use lowercase for variables in global scope? Also change the comment, we no longer have a cycle

Reply:
You mean uppercase, right?

Reply:
Yes, I'm sorry, uppercase

Reply:
I uppercased them and fixed the comment.